pom.xml

```diff
@@ -506,6 +507,16 @@
                         </execution>
                     </executions>
                 </plugin>
+
+                <!-- Version Plugin -->
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>versions-maven-plugin</artifactId>
+                    <version>${plugin.versions}</version>
+                    <configuration>
+                        <ignoredVersions>.*[-\.]M.*,.*-alpha.*,.*-beta.*,.*-RC.*,.*rc.*</ignoredVersions>
```

Comment:
@TelicentPaul Stole this from your addition to some of our other repositories

Reply:
It doesn't quite cover everything but better than nothing